vcx/wrappers/node/src/api/wallet.ts

```diff
@@ -217,9 +222,15 @@ export class Wallet {
   }
 
   /**
-   * @memberof Wallet
-   * @description Adds a record to the wallet for storage
-   * @static
+   * Adds a record to the wallet for storage
+   * Example:
+   * ```
+   * await Wallet.addRecord({
+   *    id: 'RecordId',
+   *    tags: {},
+   *    type_: 'TestType',
+   *    value: 'RecordValue'})
```

Comment:
Generally, the paradigm for readability, at least in nodejs, is to have the closing brackets 1 line below the last parameter within it. It should close like this: 
await Wallet.addRecord({
       id: 'RecordId',
       tags: {},
       type_: 'TestType',
       value: 'RecordValue'
})

Reply:
This makes it much easier to reason about and youll find nearly all node code is written this way.